portfolio/src/main/webapp/script.js

```diff
@@ -37,3 +37,16 @@ function getHellofromFetchedPractice() {
     document.getElementById('fetch-container').innerText = quote;
   });
 }
+
+/**
+ * The function is to fetch a Promise from "/data" page, convert the response message 
+ * into text and add them into the fetch-container displayed on the webpage.    
+ */
+function getDatafromJsonPractice() {
```

Comment:
Do you mean to call this function somewhere?

Reply:
This is just from step 2 practice. I just removed it.